InnerEye/ML/Histopathology/datasets/tcga_prad_dataset.py

```diff
@@ -19,44 +20,22 @@ class TcgaPradDataset(Dataset):
     - `'image_path'` (str): absolute slide image path
     - `'label'` (int, 0 or 1): label for predicting positive or negative
     """
-    SLIDE_ID_COLUMN: str = 'slide_id'
-    CASE_ID_COLUMN: str = 'case_id'
     IMAGE_COLUMN: str = 'image_path'
     LABEL_COLUMN: str = 'label'
```

Comment:
This will not work with the current dataset, discussed separately

Reply:
As agreed, I'm keeping this as an example, with a clarifying comment.

Reply:
I don't see the comment on my end but I trust it has been added